.dockerignore

```diff
@@ -0,0 +1,5 @@
+.git
+Dockerfile
+.bundle
+.env
+.idea
```

Comment:
Shouldn't `/.idea/` already be in gitignore?

Reply:
Docker doesn't use .gitignore at all 😕

Reply:
Makes sense. Nevertheless I would have thought that no `.idea` folder should be present on the build server.

Reply:
Yes, that's true, it was mostly meant for cases with local development where `docker-compose up` is used (excluding unnecessary files speeds up the initialization of the build process). But I can remove it as well.

Reply:
Done.